guest_os_booting: fix the unsupported version issues

meinaLi opened the pull request:
Fix two version issues in this PR:
    1) --reset-nvram option is not supported in RHEL8 but we need to test
       this auto case. So don't use it in the test steps.
    2) The stateless is supported from libvirt-8.6.0. So test it from this
       version.

meinaLi commented:
```
# avocado run --vt-type libvirt --test-runner=runner --vt-machine-type q35 guest_os_booting.ovmf_loader
JOB ID     : 58f64dcaf22bc302a034ca3b1f7b7fb7ab683c44
JOB LOG    : /var/lib/avocado/job-results/job-2023-06-29T03.31-58f64dc/job.log
 (1/6) type_specific.io-github-autotest-libvirt.guest_os_booting.ovmf_loader.positive_test.default_value: PASS (81.60 s)
 (2/6) type_specific.io-github-autotest-libvirt.guest_os_booting.ovmf_loader.positive_test.secure_no: PASS (83.79 s)
 (3/6) type_specific.io-github-autotest-libvirt.guest_os_booting.ovmf_loader.positive_test.stateless_yes: PASS (19.99 s)
 (4/6) type_specific.io-github-autotest-libvirt.guest_os_booting.ovmf_loader.negative_test.readonly_no: PASS (19.87 s)
 (5/6) type_specific.io-github-autotest-libvirt.guest_os_booting.ovmf_loader.negative_test.type_rom: PASS (19.55 s)
 (6/6) type_specific.io-github-autotest-libvirt.guest_os_booting.ovmf_loader.negative_test.incorrect_loader_path: PASS (17.88 s)
RESULTS    : PASS 6 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /var/lib/avocado/job-results/job-2023-06-29T03.31-58f64dc/results.html
JOB TIME   : 244.42 s
```

Yingshun commented:
> The reset-nvram is supported from libvirt-8.1.0. Here we haven't planed to test it but only use it to avoid some nvram file issues. But actually in auto job we can skip this step. So move it.

It's '8.6' in your change, could you plz clarify what version should it be?

meinaLi commented:
> > The reset-nvram is supported from libvirt-8.1.0. Here we haven't planed to test it but only use it to avoid some nvram file issues. But actually in auto job we can skip this step. So move it.
> 
> It's '8.6' in your change, could you plz clarify what version should it be?

Here I fixed two version issues. One is the --reset-nvram which is supported from libvirt-8.1.0. Another is the stateless element which is supported from libvirt-8.6.0. In this PR I added version support for the stateless but not the --reset-nvram(the description is explaining why we don't plan use --reset-nvram in this case again).

Yingshun commented:
> > > The reset-nvram is supported from libvirt-8.1.0. Here we haven't planed to test it but only use it to avoid some nvram file issues. But actually in auto job we can skip this step. So move it.
> > 
> > 
> > It's '8.6' in your change, could you plz clarify what version should it be?
> 
> Here I fixed two version issues. One is the --reset-nvram which is supported from libvirt-8.1.0. Another is the stateless element which is supported from libvirt-8.6.0. In this PR I added version support for the stateless but not the --reset-nvram(the description is explaining why we don't plan use --reset-nvram in this case again).

Can you please update commit message so others can understand the issue correctly？ BTW, I found that you just add the description in pr page, can you add them into commit too?

meinaLi commented:
@Yingshun Please help to review again.
